Fixed Grammatical mistakes on Getting Started Sections page (#2812)
umbraco/UmbracoDocs: Getting-Started/Backoffice/Sections/index.md

---

# Sections

> A section in Umbraco is where you do specific tasks related to that section. For example Content, Settings and Users are all sections. You can navigate between the different sections by clicking the corresponding icon in the section menu which is at the top of the Backoffice.

Reviewer: ```suggestion
A section in Umbraco is where you perform specific tasks related to a particular area of Umbraco  . For example Content, Settings and Users are all sections. You can navigate between the different sections by clicking the corresponding icon in the section menu which is positioned at the top of the Backoffice.
```

> ![Sections](images/highlight-sections.png "The Section menu is the horizontal menu located in the top of the Backoffice.")

Reviewer: ```suggestion
![Sections](images/highlight-sections.png "The Section menu is the horizontal menu located at the top of the Umbraco Backoffice.")
```

> *The __Section menu__ is the horizontal menu located at the top in the Backoffice.*

Reviewer: ```suggestion
*The __Section menu__ is the horizontal menu located at the top of the Umbraco Backoffice.*
```

* Grayed out nodes have not been published
* <img src="images/has-unpublished-version.svg" width="12px" style="margin: 0;"> Nodes have unpublished versions (but are currently published)

> * <img src="images/protected.svg" width="12px" style="margin: 0;"> Nodes are protected from the public (requires logging in)

Reviewer: ```suggestion
* <img src="images/protected.svg" width="12px" style="margin: 0;"> Nodes are protected from the public (can only be accessed by a logged in Member)
```

* <img src="images/locked.svg" width="12px" style="margin: 0;"> Nodes are currently locked/non-deletable

> * <img src="images/is-container.svg" width="12px" style="margin: 0;"> Nodes are containers (such as List Views)

Reviewer: ```suggestion
* <img src="images/is-container.svg" width="12px" style="margin: 0;"> Nodes are containers (a List View for example)
```

## Help sections
In the top-right corner you'll find a search tool, which is also accessible by hitting `CTRL + Space` on your keyboard.

Next to the search tool, there is a help section, where you can find Backoffice tours as well as links to Umbraco resources such as documentation and UmbracoTV.

> There's also a small user section with shortcuts to edit the user that's currently logged in, and view most recent activities.

Reviewer: ```suggestion
There is also a small 'user section' with shortcuts to edit the currently logged in user, and view their most recent activities.
```

## Access based on User Group

> Access to the section is based on which User Group a user is added to.

Reviewer: ```suggestion
Access to the section is based on which User Group a particular User belongs to.
```
